### ISSUE TITLE
Investigate error using active-standby cluster template in Liberty

### ISSUE THREAD

**Issue by jputrino** [F5Networks/f5-openstack-heat#80]
## OpenStack version
Liberty

## Heat template
active-standby cluster (from @pjbreaux's fork - https://github.com/pjbreaux/f5-openstack-heat-1/blob/feature.cluster/f5_supported/f5_plugins/active_standby.yaml)

I pasted the text of the template in via the 'Direct input' option. Here's the result:
![image](https://cloud.githubusercontent.com/assets/12143793/15831012/928dcbaa-2bd8-11e6-8b08-4cea310e11a7.png)

**Comment by jputrino** [comment]
There's an error with the cluster-ready-ve-4-nic template as well (same deployment details as above):
![image](https://cloud.githubusercontent.com/assets/12143793/15836263/31c3129e-2bf1-11e6-8291-118a78159c69.png)

**Comment by pjbreaux** [comment]
Yes this is a bug in our deployment version of Horizon in Liberty. The suggested fix is to use the python-heatclient directly. I haven't tested whether this is fixed in a newer version of Horizon for Liberty.

https://bugs.launchpad.net/horizon/+bug/1548336

**Comment by jputrino** [comment]
It seems like the easiest fix may be to just add in the defaults, as you did in the two_ve_resource_group template. I've only been including horizon instructions in the user guide/ templates, since it's so dang easy. Up to you and @swormke whether I should be including the CLI commands instead.
